### PR TITLE
fix: logic for toggle button

### PR DESCRIPTION
**Pull request by SavvyShah**
Fix for components having `multiSelect` prop.

### CODE REVIEW

File: src/utils/transform.js [hunk 1]
```diff
@@ -536,12 +538,15 @@ export const transformValueToComponentStateFormat = (value, componentProps) => {
 						if (typeof valObj === 'object' && valObj.label && valObj.value) {
 							transformedValue.push(valObj);
 						} else if (typeof valObj === 'string') {
-							const findDataObj = data.find(item => item.label.trim() === valObj.trim());
+							const findDataObj = data.find(item => item.value.trim() === valObj.trim());
```
Comment (mohdashraf010897) [inline src/utils/transform.js]: @SavvyShah User would most likely pass a `label` to select the values, because that's what is visible on the UI, revert this change. 

or if you have an explanation then please do let me know.

File: src/utils/transform.js [hunk 2]
```diff
@@ -536,12 +538,15 @@ export const transformValueToComponentStateFormat = (value, componentProps) => {
...
 							transformedValue.push(findDataObj);
 						}
 					});
+				} else if (typeof value === 'object' && value.label && value.value) {
+					transformedValue = value.value;
```
Comment (mohdashraf010897) [inline src/utils/transform.js:547]: Use label instead of value.

**Review by mohdashraf010897 — CHANGES_REQUESTED** [review]

File: src/utils/transform.js (hunk 1 above)
Comment (SavvyShah) [reply src/utils/transform.js]: The CAF and QAF values are not `Array<Label>` but `Array<Value>`. Hence it doesn't work if I change it back. Or we can change the logic inside the component itself.

File: src/utils/transform.js (hunk 2 above)
Comment (SavvyShah) [reply src/utils/transform.js:547]: Same reason as above
